include/clang/AST/ASTImporter.h

```diff
@@ -89,6 +89,8 @@ class TypeSourceInfo;
     using ImportedCXXBaseSpecifierMap =
         llvm::DenseMap<const CXXBaseSpecifier *, CXXBaseSpecifier *>;
 
+    enum class ODRHandlingType { Conservative, Liberal };
```

Comment:
In LLVM we typically name these enums as ...Kind. So it should be ODRHandlingKind.

Reply:
The description in http://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly  is not clear, in this case it is not a subclass or union type so I was not sure.

Reply:
Ok.